### ISSUE TITLE
Feature Request: Series Liquid Tag

### ISSUE THREAD

**Issue by seangwright** [forem/forem#3046]
**Is your feature request related to a problem? Please describe.**
There are other requests for features related to Series, like enhanced UX (https://github.com/thepracticaldev/dev.to/issues/2558) or a landing page for a series (https://github.com/thepracticaldev/dev.to/issues/2424).

Along with those requests to make Series more useful, I would like a way to easily link to a series from within a post.

**Describe the solution you'd like**
Using liquid tags to link to a Series based on the Series' slug. I imagine this would result in a card similar to the one rendered when linking to another post, but it would be tailored to the idea of a Series.

**Describe alternatives you've considered**
Right now when I want to link to a Series, I link to the search where the search term is the Series name.

It's a hack and not great UX, but it kinda works for Series with descriptive names.

**Additional context**
n/a

Thanks!

**Comment by Zhao-Andy** [comment]
Hey @seangwright, I think this is a good idea! We'll discuss this as a team and add some more details.

**Comment by pkfrank** [comment]
From a UI perspective, I think we could take some cues from an embedded Spotify playlist:

![spotify playlist example](https://cl.ly/b488fb96ad8d/Image%202019-06-11%20at%2011.50.26%20AM.png)

The top bar would be the Series Title, Author info, etc.  The "songs" would be one article per row.

CC @benhalpern

**Comment by jessleenyc** [comment]
Hey all! We just updated the design of series and I think it helps clarify a lot of the design aspect for a series liquid tag. Here's an example of series in action: https://dev.to/devteam/daily-challenge-43-boardgame-fight-resolver-53e3

Perhaps the liquid tag can take any article in a series and render this same series design. So if the liquid tag is {% series link %} it would know that its asking for the series embed of this collection of articles. Regardless of which link it puts from the series, the liquid tag should probably just show all the articles in order (so the user wouldn't have to find the 'first' article in the series to use the tag, they would all work).

In the codebase, series are actually referred to as collections I believe.

**Comment by GnsP** [comment]
I would like to implement the series liquid tag, if it's not already taken.

**Comment by jessleenyc** [comment]
@GnsP go for it!

**Comment by GnsP** [comment]
@jessleenyc I have opened a WIP PR for this issue, there are a couple of things that I would like to be cleared, before we proceed with this PR.

- In collections table, title of the collection is being stored as the slug and the title column is being left empty. Is this by design ? or should we refactor the codebase and migrate the existing data to store the hyphenated version of the title as slug and the title string as title ?

- What about referencing series using URL ? Right  now there is no series landing page, therefore adding support for URLs in the liquid tag is not required. However, adding support for the same would make it similar to the LinkTag and would make for better UX.
